### DISCUSSION TITLE
Skewness dan Kurtosis

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#278] [Q&A] [answered]
Hi, saya ingin bertanya mengenai konsep dari Skewness dan Kurtosis dan bagaimana cara menghitungnya di R. Terima kasih!

**Comment by anugrah18003** [comment, marked as answer]
Dalam statistik, **skewness** dan **kurtosis** adalah dua cara untuk mengukur bentuk distribusi.

**Skewness** adalah ukuran asimetri/kemiringan dari suatu distribusi. Nilai ini bisa positif atau negatif.

- Kemiringan negatif menunjukkan bahwa ekor berada di sisi kiri distribusi, yang memanjang ke arah nilai yang lebih negatif.
- Kemiringan positif menunjukkan bahwa ekor berada di sisi kanan distribusi, yang memanjang ke arah nilai yang lebih positif.
- Nilai nol menunjukkan bahwa tidak ada skewness dalam distribusi sama sekali, artinya distribusinya simetris sempurna .

**Kurtosis** adalah ukuran apakah suatu distribusi berekor berat atau berekor ringan relatif terhadap distribusi normal.

- Kurtosis dari distribusi normal adalah 3.
- Jika distribusi yang diberikan memiliki kurtosis kurang dari 3, dikatakan  playkurtic , yang berarti cenderung menghasilkan outlier yang lebih sedikit dan kurang ekstrim daripada distribusi normal.
- Jika suatu distribusi memiliki kurtosis lebih besar dari 3, dikatakan  leptokurtik , yang berarti cenderung menghasilkan lebih banyak outlier daripada distribusi normal.

Note: Data berdistribusi normal jika nilai **Skewness** bernilai 0 dan **Kurtosis** bernilai 3

Untuk menghitung skewness dan kurtosis dari dataset ini, kita dapat menggunakan fungsi **skewness()** dan **kurtosis()** dari library **moments** in R: 

library(moments)
#calculate skewness
skewness(data)

#calculate kurtosis
kurtosis(data)

Referensi:
- https://www.statology.org/how-to-report-skewness-kurtosis/
- https://www.statology.org/skewness-kurtosis-in-r/
